R/create.R

```diff
@@ -2,12 +2,17 @@
 #'
 #' @description
 #' These functions create an R project:
-#'   * `create_package()` creates an R package
+#'   * `create_package()` creates an R package.
 #'   * `create_project()` creates a non-package project, i.e. a data analysis
-#'   project
+#'     project.
+#'   * `r lifecycle::badge("experimental")` `create_quarto_project()` creates a
```

Comment:
I'm marking this as experimental because, e.g., the behaviour when called inside an existing project is not so nice. I've also exposed the bare minimum of arguments. I want to make it clear that this might very well need some refinement.